app/scripts/views/Calendar.js

```diff
@@ -77,6 +82,55 @@ class Calendar extends Component {
     )
   }
 
+  renderMap() {
+    if (!this.props.listItems.events) {
+      return null
+    } else if (this.props.listItems.events.listItems[0]) {
+      const allEvents = this.props.listItems.events.listItems
+      const allVenues = []
+      allEvents.map(item => {
+        const filterVenues = allEvents.filter(i => ((item.placeId === i.placeId)))
+        const exists = allVenues.filter(x => {
+          return x.find(({ placeId }) => placeId === item.placeId)
+        })
+        if (exists.length === 0) {
+          allVenues.push(filterVenues)
+          return allVenues
+        } return null
+      })
+      const plots = allVenues.map(item => {
+        const venueEvents = item.map(i => {
```

Comment:
I would suggest using more descriptive variable names, `i` is also usually used for `index`.